### ISSUE TITLE
can't install in fluentd 1.0.2 (dependency error)

### ISSUE THREAD

**Issue by monotek** [GoogleCloudPlatform/fluent-plugin-detect-exceptions#21]
When trying to install fluent-plugin-detect-exceptions with fluentd 1.0.2 i get the following error:

```
ERROR:  While executing gem ... (Gem::DependencyResolutionError)
    conflicting dependencies fluentd (~> 0.10) and fluentd (~> 1.0.2)
  Activated fluentd-1.0.2
  which does not match conflicting dependency (~> 0.10)

  Conflicting dependency chains:
    fluentd (~> 1.0.2), 1.0.2 activated

  versus:
    fluent-plugin-detect-exceptions (~> 0.0.8), 0.0.8 activated, depends on
    fluentd (~> 0.10)
```

**Comment by igorpeshansky** [comment]
@qingling128 1.0 should be backward-compatible with 0.14, so this may be just a matter of removing the version restriction, but we need to test that first.

**Comment by igorpeshansky** [comment]
@monotek Just curious, have to tried modifying the `Gemfile` to remove the restriction?

**Comment by monotek** [comment]
Yep, here: https://github.com/monotek/fluent-plugin-detect-exceptions/blob/772b2d2a61a719e2267a81ffae87ef3ba3f61b89/fluent-plugin-detect-exceptions.gemspec#L23

But did not work on the first try. 
I'll hope i find the time to get it testet over the weekend.

**Comment by dannyk81** [comment]
Hey @monotek, was wondering if you got this working?

Working on upgrading our fluentds to v1.0.2 from v0.12 and got stuck on this...

**Comment by monotek** [comment]
@qingling128 
building a new container with your change and fluentd 1.1.0 did not work for me:

https://hub.docker.com/r/monotek/docker-fluentd-elasticsearch/builds/baaqwyh8ynfrxtakwutcrvt/

**Comment by monotek** [comment]
Got it working now :-)
https://hub.docker.com/r/monotek/docker-fluentd-elasticsearch/builds/bahswjzprxl86wkgv7sa8qa/

**Comment by dannyk81** [comment]
Thank you @qingling128!

Will you publish a new Gem with this fix?

**Comment by qingling128** [comment]
@dannyk81 - Yup! Just released [0.0.9](https://github.com/GoogleCloudPlatform/fluent-plugin-detect-exceptions/releases/tag/v0.0.9).
